sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -3595,7 +3596,7 @@ SIREPO.app.directive('particle3d', function(appState, panelState, requestSender,
                 minZSpacing = Math.abs((zmax - zmin)) / numInterPoints;
                 var nearestIndex = 0;
                 indexMaps = [];
-                for(var i = 0; i < zpoints.length; ++i) {
+                for(i = 0; i < zpoints.length; ++i) {
```

Comment:
is this zpoints.foreach(z, i)?

Reply:
I guess so - @moellep said he prefers forEach [but...](https://stackoverflow.com/a/43032526)